### DISCUSSION TITLE
Is there a way to use the whole SD card as a hard disk?

### DISCUSSION THREAD

**Discussion by maniades** [ZuluSCSI/ZuluSCSI-firmware#24] [General]
Hi there. Is there a way to bypass the image in a file way of working, and just using the whole SD card as a single "image". That way, one could use the same SD in several different adapters, with the same behaviour in all of them (bypassing the specifics of zulu, as ini files and such). 

I would find this fallback mode quite interesting.

**Comment by aperezbios** [comment]
@maniades [Now there is](https://github.com/ZuluSCSI/ZuluSCSI-firmware/commit/8a47a017aecc6731ebb57c03764c1d373a4ab479). :) it's still experimental, but works fine in our testing. Please use https://github.com/ZuluSCSI/ZuluSCSI-firmware/releases/download/v1.0.6-rc2/ZuluSCSIv1_1_2022-08-19_53343f4.bin, which is Release Candidate 2 of firmware version 1.0.6. If no images are found, the firmware will enter a fallback mode (LED will still blink three times to indicate no images found) and will, by default, pass through the entirety of the SD card as a single SCSI disk. 

This can be overridden in your zuluscsi.ini, if you wish to expose multiple regions of sectors on the SD card as different SCSI devices. Making sure that mutiple defined regions don't overlap is entirely up to the user, at this time.  

```
# Raw sector range from SD card can be passed through
# Format is RAW:first_sector:last_sector where sector numbers can be decimal or hex.
# If end sector is beyond end of SD card, it will be adjusted automatically.
# [SCSI4]
# IMG0 = RAW:0x00000000:0xFFFFFFFF # Whole SD card
```

**Comment by maniades** [comment]
Great news!! will order one right now, and will test this as soon as it gets home. Using images is a nice approach, but I also like this way to use it. Will report as soon as I got it running.

**Comment by aperezbios** [comment]
@maniades we've just fixed a bug related to raw passthrough in firmware version 1.0.7. We recommend upgrading your firmware to version 1.0.7 at your convenience.
